lib/unleash/feature_toggle.rb

```diff
@@ -37,12 +37,11 @@ def get_variant(context, fallback_variant = Unleash::FeatureToggle.disabled_vari
 
       context = ensure_valid_context(context)
 
-      return Unleash::FeatureToggle.disabled_variant unless self.enabled && am_enabled?(context)
```

Comment:
I've shuffled this code around because if we want to be in line with the other SDKs we should still increment the variant counter for disabled